src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -280,7 +361,7 @@ static synchronized List getHostFixedTransitions(String zoneId) {
     if (idx == null) {
       return null;
     }
-    return fixedTransitions.getList(idx);
+    return transitions.getList(idx);
```

Comment:
Same comment. Would be nice to add `verifyDatabaseCached()` before it reads `transitions.